build/scripts/DownloadDotNetRuntimeInstaller.ps1

```diff
@@ -0,0 +1,43 @@
+[CmdLetBinding()]
+Param(
+    [string]$Platform = "x64",
+    [string]$OutputDirectory = ""
+)
+
+$ProgressPreference = "SilentlyContinue"
+$ErrorActionPreference = "Stop"
+
+$outputPath = Join-Path $OutputDirectory "dotnet-windowsdesktop-runtime-installer.exe"
+if(!(Test-Path $OutputDirectory))
+{
+    New-Item -ItemType Directory -Path $OutputDirectory | Out-Null
```

Comment:
Which is recommended practice?

```
New-Item BlahBlah | Out-Null
```

or

```
$null = New-Item BlahBlah
```

Reply:
Didn't know this but looks like $null is the better practice. Replacing.

Reply:
Looks like it's better to do $null